omnipaxos_ui/src/util.rs

```diff
@@ -16,7 +16,7 @@ pub(crate) mod defaults {
     pub(crate) const UI_BARCHART_GAP: u16 = 1;
     pub(crate) const UI_TABLE_CONTENT_HEIGHT: u16 = 1;
     pub(crate) const UI_TABLE_ROW_MARGIN: u16 = 1;
-    pub(crate) const COLORS: [Color; 4] = [Color::Green, Color::Blue, Color::Magenta, Color::Cyan];
+    pub(crate) const COLORS: [Color; 4] = [Color::Green, Color::Blue, Color::Cyan, Color::Magenta];
```

Comment:
Add all colors available in the library (apart from black and white) + `Color::Indexed(208)` (orange) + `Color::Indexed(211)` (pink). And try to spread them out so more contrasting colors are next to each other.